core/src/main/java/co/aikar/commands/Locales.java

```diff
@@ -118,9 +118,14 @@ public void addMessageBundles(String... bundleNames) {
     }
 
     public void addMessageBundle(String bundleName, Locale locale) {
-        if (!loadedBundles.containsEntry(bundleName, locale)) {
-            loadedBundles.put(bundleName, locale);
-            this.localeManager.addMessageBundle(bundleName, locale);
+        this.addMessageBundle(this.getClass().getClassLoader(), bundleName, locale);
+    }
+
+    public void addMessageBundle(ClassLoader classLoader, String bundleName, Locale locale) {
```

Comment:
Need to re-approach this now, what if same bundle name is in 2 different class loaders?

Reply:
I had thought about this when I wrote it, and had this idea but then it skipped over me as I was just thinking of the direct usage in a Bukkit/Bungee environment where it automatically searches for "acf-pluginname" and such. 

So something that could be potentially done is make loadedBundles a Map with the key being a ClassLoader instead, and having the value as the HashMultimap with the name of the bundle and the locale. Unless if you/anyone else has any other ideas on going around this.